docs/runbooks/SSPHighRateRejectedVms.md

```diff
@@ -0,0 +1,25 @@
+# SSPHighRateRejectedVms
+
+## Meaning
+
+This alert indicates that many Virtual Machines were not valid and therefore were not created or not modified.
+It means either a user or a script is continuously trying to create invalid Virtual Machines.
+
+## Impact
+
+There is no immediate impact, however one should see what is the reason for the large amount of rejected Virtual Machines.
+A Large amount of Virtual Machines may mean that something is not going to work as planned, for example, a script may be consistently failing to create or modify Virtual Machines)
```

Comment:
@borod108 Hi, You left the left parenthesis here. Also, not sure if it should be "consistently" or "constantly".